host/4/bookworm/powershell/powershell74/powershell74-appservice.Dockerfile

```diff
@@ -0,0 +1,65 @@
+# Build the runtime from source
+ARG HOST_VERSION=4.27.5
+FROM mcr.microsoft.com/dotnet/sdk:6.0-bookworm-slim-amd64 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest=false
+
+# Build WebJobs.Script.WebHost from source
+RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
+    git clone --branch v${HOST_VERSION} https://github.com/Azure/azure-functions-host /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
+    HOST_COMMIT=$(git rev-list -1 HEAD) && \
+    dotnet publish -v q /p:BuildNumber=$BUILD_NUMBER /p:CommitHash=$HOST_COMMIT src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj -c Release --output /azure-functions-host --runtime linux-x64 && \
+    rm -rf /azure-functions-host/workers/powershell/6 && \
+    mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
+    rm -rf /root/.local /root/.nuget /src
+
+# Install extension bundles
+RUN apt-get update && \
+    apt-get install -y gnupg wget unzip && \
+    EXTENSION_BUNDLE_VERSION_V2=2.29.0 && \
```

Comment:
Update ext bundles to be equal for all images to  : https://github.com/Azure/azure-functions-docker/blob/dev/host/4/bullseye/amd64/base/host.Dockerfile

Reply:
Done